client/ayon_maya/plugins/publish/validate_animation_product_type_publish.py

```diff
@@ -0,0 +1,42 @@
+import ayon_maya.api.action
+from ayon_core.pipeline.publish import (
+    PublishValidationError,
+    ValidateContentsOrder,
+)
+from ayon_maya.api import plugin
+
+
+class ValidateAnimationProductTypePublish(plugin.MayaInstancePlugin):
+    """Validate Animation Product Type to ensure either fbx animation
+    collector or collect animation output geometry(Alembic) enabled for
+    publishing
+    """
```

Comment:
```suggestion
    """Validate at least a single product type is exported for the instance.
    
    Validate either fbx animation collector or collect animation output 
    geometry(Alembic) enabled for publishing otherwise no products
    would be generated for the instance - publishing nothing valid.
    """
```